rep-2000.rst

```diff
@@ -0,0 +1,114 @@
+The target platforms represent the set on which all core stacks are expected to work.
+Exceptions can be made for stacks that are intentionally platform-specific.
+
+Platforms by Distribution
+=========================
+
+If planned support for a target platform is changed, notice will be sent to discourse to enable discussion of this issue.
+
+Ardent Apalone (December 2017 - December 2018)
+----------------------------------------------
+
++----------------------------------------------------------------------------------------------+
+|                                     Required support                                         |
++--------------+-----------------------+------------------------------+------------------------+
+| Architecture | Ubuntu Xenial (16.04) |     MacOS Sierra (10.12)     |   Windows 10 (VS2015)  |
++==============+=======================+==============================+========================+
+|    amd64     |          X            |             X                |          X             |
++--------------+-----------------------+------------------------------+------------------------+
+|    arm64     |          X            |                              |                        |
++--------------+-----------------------+------------------------------+------------------------+
+
+Minumum language requirements:
+
+- C++14
+- Python 3.5
+
+
+Dependency Requirements:
+
+
++---------+---------------+---------------+----------------+
+| Package | Ubuntu Xenial |     MacOS**   |   Windows 10** |
++=========+===============+===============+================+
+| CMake   |     3.5.1     |     3.11.0    |     3.10.2     |
++---------+---------------+---------------+----------------+
+| EmPY    |     3.3.2     |     3.6.5     |     3.3.2      |
++---------+---------------+---------------+----------------+
+| Poco    |     1.7.7*    |     1.7.7*    |     1.7.7*     |
++---------+---------------+---------------+----------------+
+| Python  |     3.5.1     |     3.6.5     |     3.6.4      |
++---------+---------------+---------------+----------------+
+| Ogre    |     1.10*     |     1.10*     |     1.10*      |
++---------+---------------+---------------+----------------+
+| OpenCV  |     2.4.9     |     3.4.1     |     2.4.13.2*  |
++---------+---------------+---------------+----------------+
+
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
+
+" ** " Rolling distributions will see multiple version changes of these dependencies during their lifetime.
+
+This document only captures the version at the first release of a ROS distribution and will not be updated as the dependencies move forward.
+These versions are thus a low watermark.
+
+
+Package manager use for dependencies:
+
+- Ubuntu Xenial: apt
```

Comment:
Ardent had test dependencies which came from pip. https://github.com/ros2/ros2/wiki/Linux-Development-Setup#how-to-setup-the-development-environment I dunno if they need to be listed here.

Reply:
Yeah I was wondering the same thing.In my mind this was for the "run dependencies" of the binaries we ship, so based on the binary instructions.
In which case Ardent doesn't have any runtime dependencies that need to be installed via pip (in other words you can install ROS ardent and use it without needing to have pip installed).

But the scope of this section is unclear. Maybe it should be "where we get all our dependencies" in which case pip should be listed for all platforms.